src/extract-major-beta-version.py

```diff
@@ -17,14 +17,14 @@
 
 
 def major_version_from_release_branch_name(branch_name):
-    if matches := re.match(r"^releases_v(\d+)\.0($|.0$)", branch_name):
+    if matches := re.match(r"^releases_v(\d+)$)", branch_name):
         return int(matches[1])
     raise Exception(f"Unexpected release branch name: {branch_name}")
 
 
 def get_release_branches(repo):
     return [branch.name for branch in repo.get_branches()
-            if re.match(r"^releases_v\d+\.0($|.0$)", branch.name)]
+            if re.match(r"^releases_v\d+$)", branch.name)]
```

Comment:
This is fetching the release branches from https://github.com/mozilla-mobile/firefox-android/branches which uses `releases_v109` formatting